### PR TITLE
Update cAdvisor godep

### PR DESCRIPTION
**Pull request by vmarmol**

### CODE REVIEW

File: pkg/client/containerinfo.go [hunk 1]
```diff
@@ -25,7 +25,7 @@ import (
 	"net/http"
 	"strconv"
 
-	"github.com/google/cadvisor/info"
+	info "github.com/google/cadvisor/info/v1"
```
Comment (vishh) [inline pkg/client/containerinfo.go]: nit: How about `cadvisor_api` instead?

File: pkg/client/containerinfo_test.go [hunk 2]
```diff
@@ -28,8 +28,8 @@ import (
 	"testing"
 	"time"
 
-	"github.com/google/cadvisor/info"
-	itest "github.com/google/cadvisor/info/test"
+	info "github.com/google/cadvisor/info/v1"
+	itest "github.com/google/cadvisor/info/v1/test"
```
Comment (vishh) [inline pkg/client/containerinfo_test.go]: `itest` :) How about `cadvisor_test_utils` instead?

File: pkg/client/containerinfo.go (hunk 1 above)
Comment (vmarmol) [inline pkg/client/containerinfo.go]: It is referred to as info today so it doesn't need to change :)

**Comment by vishh** [comment]
Minor nits. Overall LGTM.

File: pkg/client/containerinfo_test.go (hunk 2 above)
Comment (vmarmol) [inline pkg/client/containerinfo_test.go]: Not a fan of the name either, but this PR only updates the dep. We can change it in another PR if we'd like :)

File: pkg/client/containerinfo.go (hunk 1 above)
Comment (vishh) [inline pkg/client/containerinfo.go]: Ack.

File: pkg/client/containerinfo_test.go (hunk 2 above)
Comment (vishh) [inline pkg/client/containerinfo_test.go]: Ack.

**Comment by vishh** [comment]
LGTM

**Comment by vmarmol** [comment]
Thanks @vishh I'll send you a PR with those renames, but in Go-approved camelcase :D

**Pull request merged**